.github/workflows/gradle.yml

```diff
@@ -342,7 +344,7 @@ jobs:
         uses: chromaui/action@v1
         with:
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-          token: ${{ needs.find_valid_pat.outputs.pat }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
was there some reason we couldn't use the `find_valid_pat output` for this?

Reply:
Oh, my bad, sorry, I was just playing around with using autogenerated token

Reply:
Let me revert this back